pipeline.yml

```diff
@@ -154,7 +179,11 @@ jobs:
   serial_groups: [apps]
   plan:
   - in_parallel: *plan-gets
-  - put: deploy-dotnet-core
+  - put: deploy-dotnet-core-staging
+    params:
+      path: cf-hello-worlds/dotnet-core
+      manifest: cf-hello-worlds/dotnet-core/manifest.yml
+  - put: deploy-dotnet-core-staging
```

Comment:
Should this be `put: deploy-dotnet-core-production`?

Reply:
it does seem that way, doesn't it